lib/card.rb

```diff
@@ -1,8 +1,18 @@
-# Basic class to hold card attributes
+# Basic class to hold card attributes and calculate ordinal rank
 class Card
   attr_reader :suit, :value
+
   def initialize(suit, value)
     @suit = suit
     @value = value
   end
-end
\ No newline at end of file
+
+  def get_rank
+    face_ranks = {'K' => 13, 'Q' => 12, 'J' => 11, 'A' => 14}
+    case @value
+    when 2..10 then @value
+    else
+      face_ranks[@value]
+    end
```

Comment:
This looks a lot better but I think we could take it a step further. First of all, I don't think an instance of the Card class should care about knowing the ranks of all face cards. That seems like it should be the Deck's responsibility. Can we move the face_ranks hash accordingly? Also, since we don't expect those ranks to ever change, how about making it a constant? Finally, you've successfully consolidated the get_rank method into only two cases, so I think keeping the conditional as a case statement is overkill. In fact, case statements in general are code smells because they show that a method has too much responsibility. If you ever find yourself with so many branches of an if-statement that you feel the need to use a case statement, it's probably better to refactor the code to use a hash or move responsibilities elsewhere.